frame/bags-list/remote-tests/src/snapshot.rs

```diff
@@ -40,8 +41,6 @@ pub async fn execute<Runtime: crate::RuntimeT, Block: BlockT + DeserializeOwned>
 		.inject_hashed_prefix(&<pallet_staking::Ledger<Runtime>>::prefix_hash())
 		.inject_hashed_prefix(&<pallet_staking::Validators<Runtime>>::prefix_hash())
 		.inject_hashed_prefix(&<pallet_staking::Nominators<Runtime>>::prefix_hash())
```

Comment:
```suggestion
		.inject_hashed_prefix(&<pallet_staking::Validators<Runtime>>::map_storage_final_prefix())
		.inject_hashed_prefix(&<pallet_staking::Validators<Runtime>>::counter_storage_final_key())
		.inject_hashed_prefix(&<pallet_staking::Nominators<Runtime>>::map_storage_final_prefix())
		.inject_hashed_prefix(&<pallet_staking::Nominators<Runtime>>::counter_storage_final_key())
```

Reply:
So I'm thinking if it we could pass `fn counter_storage_final_key` to `fn inject_hashed_key` instead. 

```suggestion
		.inject_hashed_prefix(&<pallet_staking::Validators<Runtime>>::map_storage_final_prefix())
		.inject_hashed_prefix(&<pallet_staking::Nominators<Runtime>>::map_storage_final_prefix())
		.inject_hashed_key(&<pallet_staking::Validators<Runtime>>::counter_storage_final_key())
		.inject_hashed_key(&<pallet_staking::Nominators<Runtime>>::counter_storage_final_key())
```

Reply:
yes indeed